w2w/w2w.py

```diff
@@ -513,6 +513,21 @@ def wrf_remove_urban(
     newluf = luf.values.copy()
     newgreenf = greenf.values.copy()
 
+    # Based on the WRF version, defined areas to be removed:
+    if orig_num_land_cat == 61:
+        luse.values  = np.where((luse>50) & (luse<61), dst_data.ISURBAN,luse)
+        luf[dst_data.ISURBAN-1]=np.sum(luf.values[50:61,:],axis=0)
+        luf[50:61,:]=0
+    elif orig_num_land_cat == 41:
+        luse.values  = np.where((luse>30) & (luse<41), dst_data.ISURBAN,luse)
+        luf[dst_data.ISURBAN-1]=np.sum(luf.values[30:41,:],axis=0)
+        luf[30:41,:]=0
+
+
```

Comment:
I wonder whether this should be formulated more generically? In the following sense:

For the built LCZ types, W2W considers by default LCZ labels 1 to 10, see here: https://github.com/matthiasdemuzere/w2w/blob/84f7d802f8ba9b6992560cd7dbd6a503b58117b0/w2w/w2w.py#L80

This information is then stored in _Info.BUILT_LCZ_.

But, a user might also define eg. LCZ 15 (=LCZ E / Bare rock or Paved) as built-up, by providing this as an argument to the tool. 

I am not sure if:
1) anyhone has used this yet?
2) the tool is actually fully functional when using the tool?

But, perhaps this option should be considered to some extend, by using:
* _luse<=41_ and _luse<=61_?
* and using _:42_ and _:62_, instead of _:41_ and _:61_?

Does this makes sense?